doc/source/user_guide/tutorials/dpf_server/server_types.rst

```diff
@@ -29,52 +29,60 @@ using the :class:`InProcess <ansys.dpf.core.server_types.InProcessServer>` class
 
 .. rst-class:: sphx-glr-script-out
 
- .. code-block:: none
- 
-    <ansys.dpf.core.server_types.InProcessServer object at ...>
+ .. jupyter-execute::
+    :hide-code:
+
+    from ansys.dpf import core as dpf
+    local_server = dpf.start_local_server()
+    local_server
```

Comment:
These need to be calls to `print`, otherwise we just get the repr of the Server object, which looks like this:
![image](https://github.com/user-attachments/assets/ce428597-28c6-4dd0-820b-f9666046f0b2)
```suggestion
    print(local_server)
```